### PR TITLE
New post: Build a Secure To-Do App with Vue, ASP.NET Core, and Okta

### PR DESCRIPTION
**Pull request by nbarbettini**
New blog post - merge by 1/30.

### CODE REVIEW

**Comment by nbarbettini** [comment]
There seems to be a problem with rendering `{{moustaches}}` in the content (they disappear, even in source blocks). Can someone take a look and make sure it's not just my machine? Cc @jmelberg-okta

**Comment by mraible** [comment]
@nbarbettini You need raw/endraw around the code blocks:

{% raw %}
```code
```
{% endraw %}

**Comment by jmelberg-okta** [comment]
Just out of curiosity, any reason [okta-vue](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue) would't work?

**Comment by nbarbettini** [comment]
D'oh, thanks @mraible!

@jmelberg-okta This post is coming over from Scotch.io, and I wrote it before okta-vue was released. That's good idea. Let me see if I have time to update it. 👍

**Comment by nbarbettini** [comment]
A day late, but this is now ready to go. Fixed code formatting and updated the post to use okta-vue. 👍

File: _source/_posts/2018-01-31-build-secure-todo-app-vuejs-aspnetcore.md [hunk 1]
```diff
@@ -0,0 +1,1195 @@
...
+public class TodoController : Controller
+{
+    private readonly ITodoItemService _todoItemService;
+
+    public TodoController(ITodoItemService todoItemService)
+    {
+        _todoItemService = todoItemService;
+    }
+
+    // Existing code...
```
Comment (jmelberg-okta) [inline _source/_posts/2018-01-31-build-secure-todo-app-vuejs-aspnetcore.md]: nit: Missing closing bracket.

File: _source/_posts/2018-01-31-build-secure-todo-app-vuejs-aspnetcore.md [hunk 2]
```diff
@@ -0,0 +1,1195 @@
...
+
+Controllers need to opt-in to an authentication check, by adding the `[Authorize]` attribute at the top of the controller. Add this in the `TodoController`:
+
+```csharp
+[Route("api/[controller]")]
+// Add this attribute:
+[Authorize]
+public class TodoController : Controller
+{
+    // ...
```
Comment (jmelberg-okta) [inline _source/_posts/2018-01-31-build-secure-todo-app-vuejs-aspnetcore.md]: nit: Missing closing bracket

**Comment by nbarbettini** [comment]
Are you seeing something different than me @jmelberg-okta? The markdown formatting looks okay on my machine:

image

**Comment by nbarbettini** [comment]
Rebased and ready for re-review

**Comment by jmelberg-okta** [comment]
Thanks for making the updates! Looks good on my end now.

**Pull request merged**